test/unit/annotation_spec.js

```diff
@@ -68,13 +76,20 @@ describe("annotation", function () {
     }
   }
 
+  const FontDataReader = new DefaultStandardFontDataFactory({
```

Comment:
Nit: Since this is an *instance*, let's use a lower-case initial letter here.
```suggestion
  const fontDataReader = new DefaultStandardFontDataFactory({
```